### ISSUE TITLE
"TypeError: '<' not supported between instances of 'PosixPath' and 'str'" whenever using wagtail_heroicons

### ISSUE THREAD

**Issue by ReijerTheCoder** [joshuadavidthomas/wagtail-heroicons#40]
Adding `"wagtail_heroicons"` to `INSTALLED_APPS` gives the following error (on the latest version of Wagtail, on linux)

```
  File "/home/reijer/.pyenv/versions/clean-test/lib/python3.10/site-packages/wagtail/admin/views/home.py", line 388, in sprite
    return HttpResponse(icons())
  File "/home/reijer/.pyenv/versions/clean-test/lib/python3.10/site-packages/wagtail/admin/views/home.py", line 368, in icons
    all_icons = sorted(
TypeError: '<' not supported between instances of 'PosixPath' and 'str'
```

## Steps to reproduce 
- Create clean python virtual environment. (I used Python 3.10.11)
- Install wagtail `pip install wagtail`
- Create empty project `wagtail start mysite`
- In `mysite`, run migrations.
- Add `"wagtail_heroicons"` to `INSTALLED_APPS`
- `python manage.py runserver` gives the above error.

Changing line 14 of src/wagtail_heroicons/wagtail_hooks.py to read `icons.append(str(icon.path))` instead of `icons.append(icon.path)` fixes the issue.

**Comment by joshuadavidthomas** [comment]
Thanks for the detailed report. I just ran in to this myself this morning as well.

**Comment by joshuadavidthomas** [comment]
I merged in #41 and just cut a new release v0.1.3 that should be on PyPI now. Thanks!
